client/src/components/R&R/ReviewsList.jsx

```diff
@@ -30,8 +33,8 @@ const ReviewsList = ({ starFilter }) => {
   return (
     <div>
       <div className="numberAndSortType">
-        Showing {reviews.results.length}
-        reviews, sorted by
+        Showing {reviews.results.length + ' '}
+        reviews, sorted by {' '}
```

Comment:
I would put this all inside a JS snippet string to preserve formatting without the cancer-inducing spaces

Reply:
`{Showing ${reviews.results.length} reviews, sorted by}`